api/v1alpha1/envoyproxy_metric_types.go

```diff
@@ -5,10 +5,14 @@
 
 package v1alpha1
 
-type MetricSinkType string
+type (
+	MetricSinkType string
+	TraceSinkType  string
```

Comment:
since the file name is `envoyproxy_metric_types`, better to move the new struct out of this file.